src/connection/Connection.ts

```diff
@@ -272,14 +272,13 @@ export class Connection {
      * Runs all pending migrations.
      * Can be used only after connection to the database is established.
      */
-    async runMigrations(options?: { transaction?: boolean }): Promise<Migration[]> {
+    async runMigrations(options?: { transactionMode?: "all" | "none" | "each" }): Promise<Migration[]> {
```

Comment:
I think we can simplify it and just call it `transaction`

Reply:
I would prefer `transactionMode` or a variant thereof in this case. When a parameter is called `transaction` I would expect to pass an object of type `Transaction`.